Specs/Scene/MaterialSpec.js

```diff
@@ -515,8 +501,7 @@ defineSuite([
             color : new Color(0.0, 1.0, 0.0, 1.0)
         });
 
-        var pixel = renderMaterial(material1);
-        expect(pixel).toEqual([0, 255, 0, 255]);
+        renderMaterial(material1);
```

Comment:
For cases like this it's important to check the exact color, rather than only checking that it's not equal to the background color.